ui/src/pages/Upload.tsx

```diff
@@ -0,0 +1,226 @@
+import {
```

Comment:
One high level comment is that some of these field are required while others are not.

The **non required** fields are: author bio, tags and youtube link (all other fields are required)

I saw things like the youtube link being required so make sure those match what I said above

Reply:
Putting this here from Pam for reference. ![screen_shot_2021-02-06_at_4.04.38_pm.png](https://user-images.githubusercontent.com/19617248/107594001-db3c3580-6bde-11eb-926d-46b19eb20c41.png)